OpenRA.Mods.Common/Traits/AppearsOnRadar.cs

```diff
@@ -42,6 +42,14 @@ public void Created(Actor self)
 		{
 			var color = modifier != null ? modifier.RadarColorOverride(self) : self.Owner.Color.RGB;
```

Comment:
I would change the method signature on the interfaces so that you can use this as:
```csharp
var color = Game.Settings.Game.UsePlayerStanceColors ? self.Owner.PlayerStanceColor(self) : self.Owner.Color.RGB;
if (modifier != null)
    color = modifier.RadarColorOverride(self, color);
```